client/src/Components/Settings/accountForm.jsx

```diff
@@ -15,6 +16,7 @@ export default class AccountForm extends Component {
     }
 
     componentDidMount = () => {
+        axios.get()
         this.setState({
```

Comment:
I'm a little late to this party, but no reason to set props onto state. Just gets confusing as components update and props change